tests/Polykey.test.ts

```diff
@@ -0,0 +1,96 @@
+// ts imports
+import fs from 'fs'
+import os from 'os'
+import Polykey from '../src/Polykey'
+
+// js imports
+const kbpgp = require('kbpgp')
+const vfs = require('virtualfs')
+
+function randomString(): string {
+	return Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 5)
```

Comment:
This should be in our `utils` library.

Reply:
Is this a dedicated library we publish to npm? for now I just put it in `utils.ts` as we might use it later for something.